app/views/articles/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1>Listing articles</h1>
+<%= link_to 'New article', new_article_path %>
+<table>
```

Comment:
It's not a good idea to use tables, except for tabular data. Here I'd suggest using divs or article or section tags.